.github/scripts/transform_dbt_test_results.py

```diff
@@ -704,14 +728,41 @@ def main() -> None:
     workbook.save(workbook_filepath)
     print(f"Output workbook saved to {workbook_filepath}")
 
+    # Get run metadata from the environment
+    try:
+        run_by = os.environ["USER"]
```

Comment:
**thought (non-blocking):** It might be nice to set this to `GITHUB_TRIGGERING_ACTOR` if it's set.

Reply:
Excellent idea, done in [0e68fff](https://github.com/ccao-data/data-architecture/pull/370/commits/0e68fffd3238ae89c281716abe6303f98f59d9c0).